.buildkite/run-tests

```diff
@@ -27,9 +27,13 @@ run_qaf() {
 
 # ensure serverless instance is deleted even if script errors
 cleanup() {
-  echo -e "--- :elasticsearch: Tear down serverless instance $EC_PROJECT_NAME"
-  run_qaf 'qaf elastic-cloud projects delete'
-  rm -rf "$(pwd)/cloud.json"
+  if [ -z "$BUILDKITE" ]; then
+     echo -e "--- :elasticsearch: Tear down serverless instance $EC_PROJECT_NAME"
+     run_qaf 'qaf elastic-cloud projects delete'
+     rm -rf "$(pwd)/cloud.json"
```

Comment:
If we want to use `run-tests` locally, we need to tear down properly, hence the condition.